feat: add wise tutorial (#119)
beclab/docs: docs/manual/tutorials/organize-content.md

> # Build your knowledge hub using Wise

Author: ```suggestion
# Build your knowledge hub in Olares
```

Wise is a local-first, AI-powered modern reader. Seamlessly integrated with the LarePass browser extension and mobile app, it enables you to effortlessly collect, organize, and access content across platforms.

> This tutorial will guide you on how to use Wise and LarePass to collect scattered content sources and build your personal knowledge hub.

Author: ```suggestion
This tutorial guides you on how to use Wise and LarePass to collect scattered content sources and build your personal knowledge hub.
```

- Wise (built-in app)
- LarePass mobile app

## Install the LarePass browser extension

The LarePass browser extension is the core tool for content discovery and collection. Currently, only Chrome is supported.

### Install the LarePass extension 

Install the LarePass extension using one of the following ways:

- **Install from Chrome Web Store**:
  1. Search for **LarePass** in the Chrome Web Store.
  2. Open the details page, and click **Add to Chrome** to install.

- **Install offline**
  1. Visit the [LarePass page](https://www.olares.xyz/larepass) to manually download the installation file of the LarePass extension and unzip it.
  2. In the URL bar, enter `chrome://extensions/` to access the extension management page.
  3. Enable **Developer mode** in the upper right corner.
  4. Click the **Load unpacked** option, and select the unzipped LarePass extension folder to finish installing.

### Log in with your Olares account
Log into the LarePass extension by importing your Olares account:

  1. Open the LarePass extension, and click **Import an account**.
  2. Import your Olares ID using the corresponding mnemonic.
  3. Enter your Olares password to complete the login.

> You can check the **Settings** tab to ensure that the Olares is connected correctly. If it’s not connected, check whether your Olares host is accessible.

Author: ```suggestion
Check the **Settings** page to ensure that the Olares is connected correctly. If it’s not connected, check whether your Olares host is accessible.
```

## Collect web content

LarePass provides flexible and efficient ways to collect online content, including web articles, videos, podcasts, and more. 

### Collect via the LarePass extension

::: tip Uploading cookies for better experience

> Some websites restrict access for anonymous users. You can upload your cookies to Olares to improve your experience.

Author: Single-page collection is different from RSS feeds and there is no subscription involved. So this is just a recommendation. 
Actually, based on my experience, this method does not require Cookie for most of the pages.

1. Log into the website.
2. Open the LarePass extension, go to **Collect > Cookie**, and click **Upload**. You can unselect a specific cookie item by clicking the **X** button if you don't want to upload it.
:::

To collect web content using the LarePass extension:

1. Open the content page, for example, a CNN article.
2. Open the LarePass extension. The extension will automatically detect the collectable content on the page.
   ![Collect web content](/images/manual/tutorials/wise-collect-web-content.png)
3. In **Collect** > **Page**, click the <i class="material-symbols-outlined">add_box</i> button next to the title to add the page to the Wise library.

Once collected, you can find the content in **Library** > **Articles** in Wise. The audio, video, and images on the page will also be downloaded locally.

### Collect via the LarePass mobile client

> You can share web links to the LarePass mobile client for content collection. For example, to save an article from the Chrome browser on your phone:

Author: Just as an example.

1. Tap the **Share** button in the browser, then select **LarePass** as the sharing destination.
2. Open the LarePass app. LarePass will automatically detect the collectable content on the shared page and prompt whether to add it to Wise.
3. Tap **Confirm** to add it to the Wise library.

![LarePass share](/images/manual/tutorials/wise-share-to-larepass.png)

::: tip 
Alternatively, you can copy the URL directly and open LarePass. LarePass will automatically detect the URL in the clipboard and the collectable content as well.

> :::

Author: Sharing content between apps is a very common action. I think users are well aware of risk (if there is any).

3. Paste the URL. Wise will automatically detect the available RSS source.
   ![Manually add RSS](/images/manual/tutorials/wise-add-rss.png)
4. Click **Add** to complete the subscription.

### Automatically download videos

In addition to regular RSS subscriptions, you can use the LarePass extension and Wise to automatically download videos from your playlist on video platforms. Here's how to do it on YouTube:

1. Log into your YouTube account.
2. In LarePass extension, go to **Collect** > **Cookie**, and click **Upload** to upload the cookies to Olares. Olares needs the cookies of the site to access your subscription and download videos.
   
   ::: tip Enable Auto-Sync
   Cookie data may expire. You can enable the **Auto-Sync** feature in the Cookies page to ensure your cookies are updated automatically each time you visit the site.
   :::
3. Under the video you want to watch, click <i class="material-symbols-outlined">more_horiz</i> the button, select **Save**, and then click **Create New Playlist**. As an example, we created a playlist called Save to Wise.
4. Enter your playlist. The LarePass extension will automatically detect the available RSS feeds and show the **RSS** tab.

> 5. In the **RSS** tab, select the RSS feed link starting with `https://www.youtube.com/feeds/` and click <i class="material-symbols-outlined">bookmark_add</i> to subscribe.

Author: ```suggestion
5. In the **RSS** tab, select the RSS feed with a link starting with `https://www.youtube.com/feeds/...` and click <i class="material-symbols-outlined">bookmark_add</i> to subscribe.
```

    ![Subscribe to YouTube playlist](/images/manual/tutorials/wise-youtube-rss.png)

Wise will automatically download all the videos saved to this playlist.

::: tip Download videos from a specific channel
You can use LarePass extension similarly on your favorite YouTuber’s channel. Once you get the corresponding RSS feed and subscribe to it, Wise will automatically download all the videos in the channel.
:::

### Access your RSS content

To access your RSS content in Wise:
1. From the left-hand menu bar, navigate to **Subscriptions** > **Feeds**.
2. Select an unread RSS item and enter it to watch the video, listen to the podcast, or simply read the article. 

![Access RSS](/images/manual/tutorials/wise-access-rss.png)

::: tip Smart content recommendations
In addition to regular RSS subscriptions, Wise offers a fully local, 100% privacy-protected intelligent content recommendation system. You can download the recommended algorithms from the Olares app market. Learn more in [Discover themed content](../tasks/recommend.md).

> :::

Author: Good point. However, content recommendation is fundamentally different from **content collection**, which is the focus of this document. I don't want to make it a more gigantic Wise doc than it already is. That's why I make it a note.
